src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -1968,8 +1968,33 @@ void updateBuildStatus(UUID nodeUuid, BuildStatus buildStatus) {
         networkModificationTreeService.updateBuildStatus(nodeUuid, buildStatus);
     }
 
-    void invalidateBuildStatus(UUID nodeUuid, boolean invalidateOnlyChildrenBuildStatus) {
-        networkModificationTreeService.invalidateBuildStatus(nodeUuid, invalidateOnlyChildrenBuildStatus);
+    @Transactional
+    public void invalidateBuild(UUID studyUuid, UUID nodeUuid, boolean invalidateOnlyChildrenBuildStatus) {
```

Comment:
a transactional function shouldn't depend on remote services

Reply:
I agree, anyway, Transactional is not needed here. Removed.